filters/tee/teeloopback.go

```diff
@@ -0,0 +1,48 @@
+package tee
+
+import (
+	log "github.com/sirupsen/logrus"
+	"github.com/zalando/skipper/filters"
+	teePredicate "github.com/zalando/skipper/predicates/tee"
+)
+
+const FilterName = "teeLoopback"
+
+type teeLoopbackSpec struct{}
+type teeLoopbackFilter struct {
+	teeKey string
+}
+
+func (t *teeLoopbackSpec) Name() string {
+	return FilterName
+}
+
+func (t *teeLoopbackSpec) CreateFilter(args []interface{}) (filters.Filter, error) {
+	if len(args) != 1 {
+		return nil, filters.ErrInvalidFilterParameters
+	}
+	teeKey, ok := args[0].(string)
+	if !ok || teeKey == "" {
```

Comment:
Not sure why we wouldn't accept "" or no params and then use "". If somebody just playing with the routing, it may make sense.

If not however, then this can be simplified:

```
teeKey, _ := args[0].(string)
if teeKey == "" {
	return nil, filters.ErrInvalidFilterParameters
}
```

Reply:
For this one, I prefer to require a non-empty key. If we default to "" then it is more likely to end up in a scenario of circular unwanted loopbacks. WDYT?